xml/System.Net.Sockets/Socket.xml

```diff
@@ -4154,9 +4154,9 @@ This method populates the <xref:System.Net.Sockets.Socket> instance with data ga
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets or sets a <see cref="T:System.Boolean" /> value that specifies whether the <see cref="T:System.Net.Sockets.Socket" /> allows Internet Protocol (IP) datagrams to be fragmented.</summary>
+        <summary>Gets or sets a <see cref="T:System.Boolean" /> value that specifies whether the <see cref="T:System.Net.Sockets.Socket" /> sets the DF bit in the IP header, that allows Internet Protocol (IP) datagrams to be fragmented.</summary>
         <value>
-          <see langword="true" /> if the <see cref="T:System.Net.Sockets.Socket" /> allows datagram fragmentation; otherwise, <see langword="false" />. The default is <see langword="true" />.</value>
+          <see langword="true" /> if the <see cref="T:System.Net.Sockets.Socket" /> should set the DF bit in the IP Header, to forbid datagram fragmentation; otherwise, <see langword="false" />. The default is <see langword="true" />.</value>
```

Comment:
```suggestion
          <see langword="true" /> if the <see cref="T:System.Net.Sockets.Socket" /> doesn't allow datagram fragmentation; otherwise, <see langword="false" />. The default is <see langword="true" />.</value>
```